`Glob-0.10.2` fails to build on Windows with GHC 8.0 and earlier

If I try to build `Glob-0.10.2` on Windows with GHC 8.0 or earlier, it fails with:

```
[1 of 7] Compiling System.FilePath.Glob.Utils ( System\FilePath\Glob\Utils.hs, dist\build\System\FilePath\Glob\Utils.o )
System\FilePath\Glob\Utils.hs:31:1: error:
    Failed to load interface for `System.Win32.String'
    Perhaps you meant
      System.Win32.File (from Win32-2.3.1.1)
      System.Win32.Security (from Win32-2.3.1.1)
      System.Win32.Shell (from Win32-2.3.1.1)
    Use -v to see a list of the files searched for.
cabal.exe: Failed to build Glob-0.10.2 (which is required by bench:bench from text-show-3.9.4). See the build log above for details.
```

See [this CI job](https://ci.appveyor.com/project/RyanGlScott/text-show/builds/42011417/job/8rlqwnkm6js4xg13#L288) for an example. This is happening because `System.Win32.String` was introduced in `Win32-2.5.0.0`, but GHC 8.0 comes bundled with `Win32-2.3.1.1`. Can the lower version bounds on `Win32` be raised to `>= 2.5` to reflect this? Thanks!

--- a/Glob.cabal
+++ b/Glob.cabal
@@ -38,7 +38,7 @@ Library
       Build-Depends: semigroups >= 0.18
 
    if os(windows)
-      Build-Depends: Win32 >= 2
+      Build-Depends: Win32 >= 2.5
 
    Default-Language: Haskell98
 

